build.gradle

```diff
@@ -481,7 +480,7 @@ distZip {
 // simple for use in docker
 tasks.register("dockerDistUntar") {
   dependsOn distTar
-  def dockerBuildDir = "build/docker-pantheon/"
+  def dockerBuildDir = "kubernetes/"
```

Comment:
We shouldn't spew generated files into our source tree so I suspect we want this back under `build` somewhere, even if we have to copy the contents of `kubernetes` over.

Reply:
fixed, i'm using the original dir now